src/Components/Facility/Consultations/Feed.tsx

```diff
@@ -354,22 +339,23 @@ export const Feed: React.FC<IFeedProps> = ({ consultationId, facilityId }) => {
         onSuccess: async (data) => {
           if (currentPreset?.asset_object?.id && data?.position) {
             setLoading(option.loadingLabel);
-            const response = await dispatch(
-              partialUpdateAssetBed(
-                {
+            const { res, data: ResponseData } = await request(
```

Comment:
use proper variable names. 

```suggestion
            const { res, data: assetBedData } = await request(
```